gemini/use-cases/retrieval-augmented-generation/utils/intro_multimodal_rag_utils.py

```diff
@@ -673,22 +623,21 @@ def get_user_query_text_embeddings(user_query: str) -> np.ndarray:
 
 
 def get_user_query_image_embeddings(
```

Comment:
Is there a reason for this wrapper function to exist? Can `get_image_embedding_from_multimodal_embedding_model()` be called directly?

Reply:
I want to keep it modular (more than usual and expected). It helps while walking through the utils during the live sessions. I can show them the individual capabilities of the functions; in this case, I usually show a simple example of sending an image and getting embeddings (in the notebooks). 

Given that the utils have too many things going on, I can easily convey complex building blocks one at a time.